bots/image-download

```diff
@@ -42,12 +42,9 @@ import urlparse
 
 from contextlib import contextmanager
 
-BASE = os.path.normpath(os.path.dirname(os.path.realpath(__file__)))
-IMAGES = os.path.join(BASE, "images")
-DATA = os.path.join(os.environ.get("TEST_DATA", BASE), "images")
-
-# this is the path git uses within the repo, not on the filesystem (which would be IMAGES)
-GIT_IMAGE_PATH = "test/images"
+BOTS = os.path.normpath(os.path.dirname(os.path.realpath(__file__)))
```

Comment:
`os.path.dirname(os.path.realpath(__file__))` should be sufficient here
[realpath](https://docs.python.org/2/library/os.path.html#os.path.realpath) already returns the canical format